### ISSUE TITLE
Consider adding tofrodos?

### ISSUE THREAD

**Issue by ox0spy** [jlevy/the-art-of-command-line#157]
Converts text files between DOS and Unix formats.
http://manpages.ubuntu.com/manpages/raring/man1/fromdos.1.html

**Comment by Hunter-Github** [comment]
What's wrong with dos2unix, unix2dos?

**Comment by jlevy** [comment]
Yes, any thoughts on which to prefer? Apparently only fromdos supports backups. Any other significant differences in functionality?

**Comment by ShawnMilo** [comment]
It seems like a short tutorial on Perl one-liners would be even better. You can do a lot more than just replace carriage returns. I gave up on Perl _long_ ago in favor of Python, but I still use it all the time for one-liners.

**Comment by jlevy** [comment]
@ShawnMilo if you have ideas on #115 feel free to give comments/suggestions there.
